src/server/request.rs

```diff
@@ -9,7 +9,7 @@ use std::time::Duration;
 use buffer::BufReader;
 use net::NetworkStream;
 use version::{HttpVersion};
-use method::Method::{self, Get, Head};
+use method::Method::{self};
```

Comment:
So this last '::{self}' part can be removed now.

Hmm in this diff it seems that this use statement can be removed.